### PR TITLE
feature: boost phrase and distance phrase boosting for title and comments

### PR DESCRIPTION
**Pull request by cdxker**

### CODE REVIEW

File: hackernews-ingest/bulk_ingest/bulk_send.yaml [hunk 1]
```diff
@@ -27,3 +27,12 @@ spec:
...
           - name: API_BASE_URL
             value: "https://hackernews.withtrieve.com/api"
+          - name: "COMMENT_DISTANCE_FACTOR"
+            value: 1.3
+          - name: "COMMENT_BOOST_FACTOR"
+            value: 1.3
+          - name: "STORY_DISTANCE_FACTOR"
+            value: 1.5
+          - name: "STORY_BOOST_FACTOR"
+            value: 1.5
```
Comment (skeptrunedev) [inline hackernews-ingest/bulk_ingest/bulk_send.yaml]:
```suggestion
          - name: "COMMENT_DISTANCE_FACTOR"
            value: 0.25
          - name: "STORY_BOOST_FACTOR"
            value: 1.38
```

**Review by skeptrunedev — CHANGES_REQUESTED** [review]
No distance for stories or boost for comments

**Review by skeptrunedev — CHANGES_REQUESTED** [review]
- [x] text of show posts 
- [x] show tag

**Comment by cdxker** [comment]
Just added the requested changes

File: hackernews-ingest/bulk_ingest/bulk_send.yaml (hunk 1 above)
Comment (cdxker) [reply hackernews-ingest/bulk_ingest/bulk_send.yaml]: Were these values tested for jina or bge-m3?

**Review by skeptrunedev — APPROVED** [review]

**Pull request merged**